code/edgeconsent/build.gradle

```diff
@@ -174,12 +141,12 @@ publishing {
                     }
                 }
                 scm {
-                    connection = 'scm:git:github.com//Adobe-Marketing-Cloud/acp-sdks.git'
-                    developerConnection = 'scm:git:ssh://github.com//Adobe-Marketing-Cloud/acp-sdks.git'
-                    url = 'https://github.com/Adobe-Marketing-Cloud/acp-sdks'
+                    connection = 'scm:git:github.com//adobe/aepsdk-edgeconsent-android.git'
+                    developerConnection = 'scm:git:ssh://github.com//adobe/aepsdk-edgeconsent-android.git'
+                    url = 'https://github.com/adobe/aepsdk-edgeconsent-android'
                 }
                 withXml {
-                    def dependenciesNode = asNode().appendNode('dependencies')
+                    def dependenciesNode = asNode.appendNode('dependencies')
```

Comment:
I believe this warning was just saying asNode() uses default null string which did not seem like a problem, does asNode return same object here?